airflow/dag_processing/manager.py

```diff
@@ -580,6 +580,7 @@ def _run_parsing_loop(self):
                     pass
                 elif isinstance(agent_signal, CallbackRequest):
                     self._add_callback_to_queue(agent_signal)
+                    self.log.debug("_add_callback_to_queue; agent signal; %s", agent_signal)
```

Comment:
I don’t have a strong opinion, but do we need to commit this log, or is it only for testing?